test/StoragelessTest/Http/SessionMiddlewareTest.php

```diff
@@ -53,30 +51,27 @@
 
 final class SessionMiddlewareTest extends TestCase
 {
-    public function testFromSymmetricKeyDefaultsUsesASecureCookie() : void
-    {
-        $response = SessionMiddleware::fromSymmetricKeyDefaults('not relevant', 100)
-            ->process(new ServerRequest(), $this->writingMiddleware());
-
-        $cookie = $this->getCookie($response);
-
-        self::assertTrue($cookie->getSecure());
-        self::assertTrue($cookie->getHttpOnly());
-    }
-
-    public function testFromAsymmetricKeyDefaultsUsesASecureCookie() : void
+    /**
+     * @see https://tools.ietf.org/html/rfc6265#section-4.1.2.5 for Secure flag
+     * @see https://tools.ietf.org/html/rfc6265#section-4.1.2.6 for HttpOnly flag
+     * @see https://github.com/psr7-sessions/storageless/pull/46 for / path
+     * @see https://tools.ietf.org/html/draft-ietf-httpbis-cookie-same-site for SameSite flag
+     * @see https://tools.ietf.org/html/draft-ietf-httpbis-cookie-prefixes for __Secure- prefix
+     *
+     * @dataProvider defaultMiddlewaresProvider
+     * @group #46
+     */
+    public function testDefaultMiddlewareConfiguresASecureCookie(SessionMiddleware $middleware) : void
```

Comment:
I've grouped all the cookie security related assertions into one test, hope the tests refactor is ok for you